rocketmq-streams-examples/README.md

```diff
@@ -0,0 +1,67 @@
+## rocketmq-streams-examples
```

Comment:
With these samples, I still don't know how to construct the input or what expected results I can get.